database/migrations/2021_03_01_100000_base_permissions_migration.php

```diff
@@ -6,22 +6,16 @@
 use Spatie\Permission\Contracts\Permission;
 use Spatie\Permission\PermissionRegistrar;
 use Spatie\Permission\Models\Role;
-use Illuminate\Support\Collection;
 
-class CreatePermissionsMigration extends Migration
+class BasePermissionsMigration extends Migration
```

Comment:
Better to put this class somewhere in the `src\...` tree so its properly name spaced and  auto-loadable by other packages (this class itself isn't a migration).

Reply:
You learn something new everyday working with a senior dev.  I'm on it.